libvast/vast/system/import_command.hpp

```diff
@@ -204,53 +180,71 @@ import_command(const command::invocation& invocation, caf::actor_system& sys) {
   auto guard = system::signal_monitor::run_guarded(
     sig_mon_thread, sys, defaults::system::signal_monitoring_interval, self);
   // Get node components.
-  auto components = get_node_components(self, node, {"accountant", "importer"});
+  auto components = get_node_components(
+    self, node, {"accountant", "type-registry", "importer"});
```

Comment:
Can you add an assertion that the return set size is big enough so that the index access later on will be guaranteed to be valid?